object-filter/src/main/java/org/infinispan/objectfilter/impl/predicateindex/be/PredicateNode.java

```diff
@@ -57,7 +57,7 @@ public void handleChildValue(BENode child, boolean childValue, FilterEvalContext
 
       if (isEvaluationComplete(evalContext)) {
          if (predicate.isRepeated() && evalContext.treeCounters[startIndex] == value) {
-            // receiving the same value multiple times if fine if this is a repeated condition
+            // receiving the same value multiple times is fine if this is a repeated condition  //todo ?????
```

Comment:
TBH I do not remember now what I meant with that comment, I just fixed the spelling and then wondered if there was something to be fixed there. I'll remove the TODO.